services/githubService.js

```diff
@@ -156,10 +163,39 @@ const fetchOpenPRs = async (perPage = 10, page = 1) => {
   }
 };
 
+/**
+ * Fetches issues across all repositories in the ORG
+ */
+const fetchIssues = async () => {
+  try {
+    const baseURL = config.get("githubApi.baseUrl");
+    const issues = "/issues";
+    const urlObj = new URL(baseURL);
+    urlObj.pathname = "orgs" + "/" + ORG + issues;
+    const createdURL = urlObj.href;
+    const res = await getFetchWithAuthToken(
+      createdURL,
+      {
+        filter: "all",
+      },
+      {
+        Accept: "application/vnd.github+json",
+        Authorization: `Bearer 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`,
```

Comment:
Why do we have a token here?

Reply:
This token is used to fetch issues from RDS org as an authenticated user. This token has been revoked. I've added it here so that the org owner can replace this token easily with PAT token from .env file once it is created.

Reply:
I suggest we store this token as an environment variable and import it from there